qemu/tests/x86_cpu_model.py

```diff
@@ -47,11 +56,12 @@ def run(test, params, env):
     else:
         test.cancel("This host doesn't support cpu model %s" % model)
 
-    params["paused_after_start_vm"] = "no"
     params["cpu_model"] = cpu_model
+    params["start_vm"] = "yes"
```

Comment:
You did not create vm before this line, so can we remove it?

Reply:
No, we can't. It overrides the param "start_vm" in cfg to let VM start.